electrical/wiring/README.md

```diff
@@ -5,6 +5,8 @@ The wiring is what connects each of the 10 motors to the motor board carrying po
 ![wiring diagram](images/wiring_diagram.png)
 Wiring diagram for one side of the rover. The other side is identical.
 
+Harness diagrams further below were generated with [WireViz](https://github.com/wireviz/WireViz/) and can be re-generated with the *generateHarness.bat/.sh* scripts.
```

Comment:
This would be a good place to explain to the uninitiated what a wiring harness.

Reply:
Love the feedback! Next commit will have a note on terminology that explains the wiring harness.

Reply:
Commit db7e185